src/instructlab/model/evaluate.py

```diff
@@ -335,7 +335,15 @@ def launch_server(
     try:
         # http_client is handling tls params
         api_base = backend_instance.run_detached(
-            http_client(ctx.params), background=not enable_serving_output
+            http_client(
```

Comment:
What's the reason this change was included but not one for `run_detached()` in data/generate.py?

Reply:
MyPy didn't complain about this change needing to be made there, but we should definitely update it there as well.